docs/manual/docs/cmd/spo/listitem/listitem-record-undeclare.md

```diff
@@ -0,0 +1,44 @@
+# spo listitem record undeclare 
+
+Undeclares listitem as a record
+
+## Usage
+
+```sh
+spo listitem record undeclare [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|URL of the site where the list item should be undeclared as a record
+`-i, --id <id>`|ID of the list item to be undeclared as a record.
+`-l, --listId [listId]`|ID of the list where the list item should be undeclared as a record. Specify listId or listTitle but not both
+`-t, --listTitle [listTitle]`|Title of the list where the list item should be undeclared as a record. Specify listId or listTitle but not both
+`-o, --output [output] `|Output type. json|text. Default text
+`--verbose `|Runs command with verbose logging
+`--debug `| Runs command with debug logging
+
+!!! important
+    Before using this command, log in to a SharePoint Online site, using the spo login command.
```

Comment:
For consistency, `spo login` should be formatted as link to the help page of the command